robot/EDA/tests/browser/contacts/verify_preferred_phone.robot

```diff
@@ -0,0 +1,166 @@
+*** Settings ***
+Documentation
+...     Verify Preferred Phone functionality
+
+
+Resource        robot/EDA/resources/EDA.robot
+Library         DateTime
+Library         SeleniumLibrary
+Library         cumulusci.robotframework.PageObjects
+...             robot/EDA/resources/ContactsPageObject.py
+...             robot/EDA/resources/TriggerHandlersPageObject.py
+Suite Setup     Run keywords
+...             Initialize test data
+...             Open test browser and print package details
+Suite Teardown  Capture screenshot and delete records and close browser
+
+*** Test Cases ***
+Verify basic preferred phone functionality
+    Select app launcher                     EDA
+    Close all tabs
+
+    Current page should be                  Home
+    ...                                     Contacts
+
+    Open EDA Settings Tab menu item
+    Enable enchanced checkbox
+    Clear the disable preferred phone enforcement
+    Shift to default content
+
+#    Enable trigger handler
+
+#    Current page should be                  Home
+#    ...                                     Contacts
+    Select tab                              Contacts
+    Select contact                          ${CONTACT.FirstName}
+    ...                                     ${CONTACT.LastName}
+    Validate preferred phone form
+
+Verify disable preferred phone enforcement
+    Select app launcher                     EDA
+    Close all tabs
+    Current page should be                  Home
+    ...                                     Contacts
+
+    Open EDA Settings Tab menu item
+    Enable enchanced checkbox
+    Verify setting of disable preferred phone enforcement
+    Shift to default content
+
+#    Close all tabs
+#    Current page should be                  Home            
+#    ...                                     Contacts
+    Select tab                              Contacts
+    Select contact                          ${CONTACT.FirstName}
+    ...                                     ${CONTACT.LastName}
+
+    Test home phone functionality           ${CONTACT.FirstName}
+    ...                                     ${CONTACT.LastName}
+    Sleep                                   1
+
+Verify batch functionality of preferred phone
+    Select app launcher                     EDA
+    Close all tabs
+
+    Current page should be                  Home           Contacts
+
+    # Verify the EDA Setting 'Disable Preferred Phone enforcement'
+    
+    # The following places us inside an iFrame.  Remember to jump 
+    # back out when done.
+
+    # Make sure the 'Disable Preferred Phone enforcement' checkbox 
+    # has a check in the checkbox
+    Open EDA Settings Tab menu item
+    Enable enchanced checkbox    
+    Set the disable preferred phone enforcement
+    Disable enchanced checkbox    
+    Shift to default content
+
+    # Clear the 'Active' checkbox
+#    Load page object                        Home            TriggerHandlers
```

Comment:
Why is this commented out? If it's not necessary, I recommend removing it.

Reply:
All of this code was already cleaned up